pkg/iostreams/color.go

```diff
@@ -94,6 +94,10 @@ func (c *ColorScheme) Gray(t string) string {
 	return gray(t)
 }
 
+func (c *ColorScheme) Grayf(t string, args ...interface{}) string {
```

Comment:
Is this just for convenience so that you don't have to use `fmt.Sprintf`? Should it exist for only some colors but not for others?

Reply:
Purely for convenience. I can add it for other colors.